From 5b39b3e76ed98542cdaa5a9af901691023da07dc Mon Sep 17 00:00:00 2001
From: Tianshu Qiu <tian.shu.qiu@intel.com>
Date: Tue, 16 Dec 2014 11:23:04 +0800
Subject: [PATCH 191/196] atomisp2: Set stream as online when using YUVPP pipe

When using YUVPP pipe, the online field in stream config
should be set as true.

Change-Id: Ia7903b9b6ae285581f18e06ce50628ef970be508
Tracked-On: https://jira01.devtools.intel.com/browse/IMINAN-13815
Signed-off-by: Tianshu Qiu <tian.shu.qiu@intel.com>
Reviewed-on: https://icggerrit.ir.intel.com/32925
Reviewed-by: Jing Tao <jing.tao@intel.com>
Reviewed-by: Bin B Han <bin.b.han@intel.com>
Approver: Wei Tao <wei.tao@intel.com>
Reviewed-by: Wei Tao <wei.tao@intel.com>
Build: Wei Tao <wei.tao@intel.com>
Integrator: Wei Tao <wei.tao@intel.com>
Maintainer: Wei Tao <wei.tao@intel.com>
---
 .../drivers/media/pci/atomisp2/atomisp_driver/atomisp_compat_css20.c     | 1 +
 1 file changed, 1 insertion(+)

diff --git a/drivers/external_drivers/camera/drivers/media/pci/atomisp2/atomisp_driver/atomisp_compat_css20.c b/drivers/external_drivers/camera/drivers/media/pci/atomisp2/atomisp_driver/atomisp_compat_css20.c
index 7aa5b96..86b5958 100644
--- a/drivers/external_drivers/camera/drivers/media/pci/atomisp2/atomisp_driver/atomisp_compat_css20.c
+++ b/drivers/external_drivers/camera/drivers/media/pci/atomisp2/atomisp_driver/atomisp_compat_css20.c
@@ -2216,6 +2216,7 @@ void atomisp_css_enable_continuous(struct atomisp_sub_device *asd,
 	 */
 	if (ATOMISP_USE_YUVPP(asd)) {
 		stream_env->stream_config.continuous = 0;
+		stream_env->stream_config.online = 1;
 		return;
 	}
 
-- 
1.9.1

